nucypher/policy/conditions/time.py

```diff
@@ -33,14 +37,19 @@ def __init__(
         return_value_test: ReturnValueTest,
         chain: int,
         method: str = METHOD,
+        condition_type: str = ConditionType.TIME.value,
```

Comment:
Same question for `CompoundAccessControlCondition` and `RPCCondition`

Reply:
Sorry, I missed your question @manumonti. 

Yes, it seems to be an improvement. I think I originally did that because `TypedDict` (which is used in serialization?) expects all fields to be of primitive type. I will reproduce this issue and post it here. Maybe there is a way around that.